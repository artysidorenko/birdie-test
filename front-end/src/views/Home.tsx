import * as React from 'react';

const Dashboard = () => (
  <>
    Welcome to the Care Dashboard.
    Here you can view information about the care being received by the receipient,
    including updates on their mood and health.
  </>
);

export default Dashboard;
