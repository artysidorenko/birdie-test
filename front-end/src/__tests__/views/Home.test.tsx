import * as React from 'react';
import { shallow } from 'enzyme';
import { expect as _expect } from 'chai';

import Home from '@App/views/Home';

/* Normally part of src/setupTests.js but ... */
import { configure } from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
/* ... file was not being picked up by CRA so added manually */

/* tslint:disable:no-unused-expression */

describe('Home Page', () => {

  it('renders without crashing (shallow)', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
