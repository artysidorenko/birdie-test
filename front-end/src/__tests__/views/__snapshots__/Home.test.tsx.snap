// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders without crashing (shallow) 1`] = `
<Fragment>
  Welcome to the Care Dashboard. Here you can view information about the care being received by the receipient, including updates on their mood and health.
</Fragment>
`;
